.github/workflows/ruby.yml

```diff
@@ -22,3 +22,5 @@ jobs:
         bundle --jobs 4 --retry 3
     - name: Run tests
       run: bundle exec rspec gilded_rose_spec.rb
+    - name: Run mutant
+      run: bundle exec mutant --ignore-subject GildedRose.update_quality -- GildedRose
```

Comment:
I'm almost certain I need to use `'GildedRose*'` here instead.

Reply:
Yes, else it does not recurse and only selects subjects class/instance methods directly on that class.